pytube/streams.py

```diff
@@ -57,7 +57,10 @@ def __init__(
         self.video_codec, self.audio_codec = self.parse_codecs()
 
         self.is_otf: bool = stream["is_otf"]
-        self.bitrate: Optional[int] = stream["bitrate"]
+        if "bitrate" in stream.keys():
+            self.bitrate: Optional[int] = stream["bitrate"]
+        else:
+            self.bitrate = 0
```

Comment:
I believe using the default if not found in stream dict will do the same cleaner.

```suggestion
        self.bitrate = stream.get("bitrate", 0)
```